packages/@aws-cdk/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -27,6 +27,13 @@ export interface EcsRunTaskProps extends sfn.TaskStateBaseProps {
    */
   readonly taskDefinition: ecs.TaskDefinition;
 
+  /**
+   * Can specify revision number of ECS task definiton family
+   *
+   * @default - None specify revision number. for this reason you will be specifying latest.
+   */
+  readonly revisionNumber?: number;
```

Comment:
```suggestion
  /**
   * The revision number of ECS task definiton family
   *
   * @default '$latest'
   */
  readonly revisionNumber?: number;
```

If it's not `'$latest', please specify it in the docstring